src/main/java/com/hubspot/jinjava/objects/collections/PyMap.java

```diff
@@ -21,6 +21,13 @@ protected Map<String, Object> delegate() {
     return map;
   }
 
+  public Object get(String key, Object defaultValue) {
+    if (!map.containsKey(key)) {
+      return defaultValue;
+    }
+    return map.get(key);
```

Comment:
```suggestion
    return getOrDefault(key, defaultValue);
```

Can just proxy to the existing method